support-frontend/assets/components/page/page.jsx

```diff
@@ -31,6 +31,15 @@ export default function Page(props: PropTypes) {
     </div>
   ) : null;
 
+    React.useEffect(() => {
+      // setTimeout(() => {
+        if (window.location.hash) {
+          const hashElement = document.getElementById(window.location.hash.substr(1));
+          if (hashElement) { hashElement.scrollIntoView(); }
+        }
+      // }, 1);
```

Comment:
this 1ms timeout is needed, otherwise this happens
![notenough](https://user-images.githubusercontent.com/7304387/89528008-36c9d880-d7e2-11ea-9e62-479a9a596460.gif)

Reply:
If it just needs to be deferred for a tick after render, maybe try setImmediate? Or if it needs more than a tick, requestAnimationFrame. It would be nice to have something that's a bit more obvious in its intent than a setTimeout with a magic number. I am not 100% sure on how React's rendering process works though, so I don't know if there's a better way to pinpoint when the target component renders.

Reply:
setImmediate has the same behaviour as the screenshot above!
requestAnimationFrame worked though.
I will update the PR, thanks for your help!